docs/developer-guide/processing-log.rst

```diff
@@ -96,10 +96,10 @@ message (STRUCT)
 
 message.type (INT)
   An int that describes the type of the log message. Currently, the following types are
-  defined: 0 (DESERIALIZATION_ERROR), 1 (RECORD_PROCESSING_ERROR).
+  defined: 0 (DESERIALIZATION_ERROR), 1 (RECORD_PROCESSING_ERROR), 2 (PRODUCTION_ERROR).
```

Comment:
Can you update the DDL on line 198?

Reply:
Done -- good catch! Also added the new flag to the config reference docs (and fixed a few errors while I was in there).